internal/a3m/a3m.go

```diff
@@ -165,6 +168,14 @@ func (a *CreateAIPActivity) Execute(
 	return result, nil
 }
 
+// taskStatusesByA3mJobStatuses maps a3m job statuses to Enduro task statuses.
+var taskStatusesByA3mJobStatuses = map[transferservice.Job_Status]enums.TaskStatus{
```

Comment:
I think the name we better before, or what I'd understand better, `a3mJobToTaskStatuses`. Not a big deal for sure.